app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+<div class="top_login">
+<%= link_to '<', root_path, class:"back_logo" %>
+<h2>Sign up</h2>
+<%= f.submit "Sign up" %>
+</div>
+
+  
+<div class="body_login">
+  <div class="field">
+    <%= f.text_field :name, autofocus: true, placeholder: "Name", class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autocomplete: "email", placeholder: "Email",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+  
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Password confirmation",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= render "devise/shared/error_messages", resource: resource %>
+  </div>
+<% end %>
+</div>
```

Comment:
- [ ] For the registration page, there are some things left to do as well:
  This is how it looks in your design:
  
  ![image](https://user-images.githubusercontent.com/20567503/209024408-7e64067f-afac-4a66-af98-e1f0a487ccff.png)

  And this is how it should look:
  
  ![image](https://user-images.githubusercontent.com/20567503/209024490-73e0a156-c190-4a6e-bc21-e32834ea25ae.png)
  
  Kindly match your design with the original design, by fixing the signup button's style and removing the input text field that's left.